test/__snapshots__/ruby.test.js.snap

```diff
@@ -97,78 +97,133 @@ exports[`for the off.json config assign.rb matches expected output 1`] = `
 "# frozen_string_literal: true
 
 # rubocop:disable Lint/UselessAssignment, Style/ParallelAssignment
-
-a = 1
-
-a =
-  begin
-    2
+# rubocop:disable Metrics/MethodLength
```

Comment:
This is great @johnschoeman! Could you split up these tests into multiple methods? That way they're easier to understand and when one of the assertions fails it'll be easier to find it.